### ISSUE TITLE
Recurring Payments: Voiceover Cursor Focus on Payment Form

### ISSUE THREAD

**Issue by donalirl** [Automattic/jetpack#15705]
I am opening this up on behalf of a visually-impaired user who uses Apple Voiceover,

#### Steps to reproduce

1. Using a screen reader such as Apple Voiceover, navigate to and tap on the payment button created with a Recurring Payments block.
2. The credit card payment form opens up — attempt to start filling this in.

#### What I expected

I should be able to tap on the payment button, and the cursor should be focused at the top of the page so I can begin swiping right (using a screen reader) to enter fields.

#### What happened instead

The voiceover cursor is not focused on the form so it's not possible to start entering the credit card details into the form.

#### Browser / OS version

Device: iPod Touch
Browser: Safari Latest
Screen reader: Apple Voiceover

#### Screenshot / Video

https://d.pr/i/BIrmeM

#### Context / Source

#p4vt7e-6n-p2

#journey #empathy #livesharing #accessibility #painpoint #user-report

**Comment by valeciawolf34** [comment]
https://support.google.com/contributionpolicy/answer/7422880

**Comment by valeciawolf34** [comment]
https://drive.google.com/file/d/1jBLGy7sfFu9ILJI2m8Wagrr3rXOa7ZYG/view?usp=drivesdkc+ZQKQwTKR4BWV+WelL2fZ3mY5bo3Kb0Yd/H66ktZk0=

Valecia G Rodriguez

**Comment by valeciawolf34** [comment]
Valecia G Rodriguez

**Comment by valeciawolf34** [comment]
Valecia G Rodriguez
